src/vstart.sh

```diff
@@ -378,6 +380,16 @@ case $1 in
         pci_id="$2"
         spdk_enabled=1
         shift
+	;;
+    --bluestore-devs )
+	IFS=',' read -r -a bluestore_dev <<< "$2"
+	for dev in "${bluestore_dev[@]}"; do
+	    if [ ! -b $dev ]; then
```

Comment:
Is it worthwhile also adding a `-w` test as well to catch permission errors early.

Reply:
Good idea.